### PR TITLE
update category and normalise enums

### PR DESCRIPTION
**Pull request by ayushthe1**
**Description**

This PR : 
* Changes `Orchestration & Management` category to `Service Mesh`
* Normalize model-name to smp-enum (ISTIO) and displayName to Istio

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: build/config.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ const Component = "Istio"
 var Meshmodelmetadata = make(map[string]interface{})
 
 var MeshModelConfig = adapter.MeshModelConfig{ //Move to build/config.go
-	Category: "Orchestration & Management",
+	Category: "Service Mesh",
```
Comment (leecalcote) [inline build/config.go:30]:
```suggestion
	Category: "Cloud Native Network",
```

**Review by leecalcote — CHANGES_REQUESTED** [review]
@ayushthe1 thank you. Could we switch to this category name instead?

File: build/config.go (hunk 1 above)
Comment (ayushthe1) [reply build/config.go:30]: ok @leecalcote , i will fix it .

Comment (leecalcote) [reply build/config.go]: Thank you, sir.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**